client/me/profile-gravatar/index.jsx

```diff
@@ -25,6 +27,18 @@ export default React.createClass( {
 	render() {
 		const profileURL = `//gravatar.com/${ this.props.user.username }`;
 
+		if ( isEnabled( 'me/edit-gravatar' ) ) {
+			return (
+				<div className="profile-gravatar">
+					<Gravatar user={ this.props.user } size={ 150 } imgSize={ 400 } />
```

Comment:
I'm not 100% clear on why 400 is used, in this case I'm continuing to use the same imgSize that was used before in the same place.

From a search through the codebase, it looks like we are using imgSize=400 for larger images (larger than the default 96) for caching purposes. Adding that in as a comment.